.github/workflows/release.yml

```diff
@@ -24,20 +20,34 @@ jobs:
       - name: Update Npm
         run: npm install -g npm@latest
 
+      # Increment version to next major version.
+      # Store version as an environment variable for later steps to use.
+      - name: Set Version of this Release
+        run: |
+          npm version major
+          cd example
+          npm version major
```

Comment:
According to the aforementioned diagram, we want to increase the second number, i.e. from 1.0.1 to 1.1.0. Doing `npm version major` would increase it from 1.0.1 to 2.0.0. `npm version minor` accomplishes what we want per the diagram.